cmd/gateway-azure.go

```diff
@@ -181,6 +156,66 @@ func azureToObjectError(err error, params ...string) error {
 	return e
 }
 
+// mustGetAzureUploadID - returns new upload ID which is hex encoded 8 bytes random value.
+func mustGetAzureUploadID() string {
+	var id [8]byte
+
+	n, err := io.ReadFull(rand.Reader, id[:])
+	panic(fmt.Errorf("unable to generate upload ID for azure. %s", err))
```

Comment:
This might panic all the time @balamurugana if will have to do `if err != nil` on this one.

Reply:
Done